.github/workflows/build-ci.yml

```
@@ -85,15 +94,24 @@ jobs:
         with:
           java-version: '8.0.292'
 
+      - name: Get Semver
+        id: semver
+        if: ${{ startsWith(github.ref, 'refs/heads/release/v') }}
+        run: |
+          semver=$(echo ${{ github.ref }} | tail -c +21)
+          echo "::set-output name=semver::$semver"
+
       - name: Docker meta
         id: docker_meta
         uses: docker/metadata-action@v3.3.0
         with:
           images: ghcr.io/${{ github.repository_owner }}/grpc-proxy
+          flavor: |
+            latest=false
           tags: |
             type=edge,branch=main
-            type=semver,pattern={{version}}
-            type=semver,pattern={{major}}.{{minor}}
+            type=raw,priority=950,priority=950,enable=${{ startsWith(github.ref, 'refs/heads/release/v') }},value=${{ steps.semver.outputs.semver }}
```

Duplicate `priority` attribute
```suggestion
            type=raw,priority=950,enable=${{ startsWith(github.ref, 'refs/heads/release/v') }},value=${{ steps.semver.outputs.semver }}
```